vulcanai2/datasets/tabulardataset.py

```diff
@@ -241,37 +259,51 @@ def create_label_encoding(self, column_name, ordered_values):
         if set(column_vals) != set(self.list_all_column_values(column_name)):
             raise ValueError("Not all column values are included")
 
-        self.df[column_name] = getattr(self.df, column_name).map(ordered_values)
+        self.df[column_name] = getattr(self.df, column_name).map(
+            ordered_values)
 
         logger.info(f"Successfully remapped {column_name}")
 
     def create_one_hot_encoding(self, column_name, prefix_sep="@"):
         """
-        Create one-hot encoding for the provided column. Deliberatly doesn't make any decisions for you.
-        :param column: The name of the column you want to one-hot encode
-        :param prefix_sep: The prefix seperator.
-        :return: None
+        Create one-hot encoding for the given column.
+
+        Parameters
+        ----------
+        column_name: String
+            The name of the column you want to one-hot encode
+        prefix_sep String default("@")
+            The prefix used when creating a one-hot encoding
         """
-        #TODO: ensure dummy_na =False is what you want
+
+        # TODO: ensure dummy_na =False is what you want
         if column_name in list(self.df):
```

Comment:
if you're providing a column name to encode, why loop through?

Reply:
Good point, I'll let it fail through pandas